server/src/main/java/org/cloudfoundry/identity/uaa/scim/vmidentity/VmidentityScimGroupProvisioning.java

```diff
@@ -33,29 +35,30 @@
 import com.vmware.identity.idm.PrincipalId;
 import com.vmware.identity.idm.client.CasIdmClient;
 
-public class VmidentityScimGroupProvisioning implements ScimGroupProvisioning, ScimGroupExternalMembershipManager {
+public class VmidentityScimGroupProvisioning implements ScimGroupProvisioning, SystemDeletable {
 
-    private final CasIdmClient _idmClient;
+    public static final String GROUP_FIELDS = "id,displayName,description,created,lastModified,version,identity_zone_id";
```

Comment:
Unfortunately - yes, it's the set of fields we're allowed to perform ordering with. It gets used as part of a validation.